docs/store/filecoin-plus.md

```diff
@@ -0,0 +1,113 @@
+- Follow operational guidelines, keep a record of decision flow, and respond to any requests for audits of their allocation decisions.
+
+You can find a list of current [active notaries at plus.fil.org](https://plus.fil.org).
+
+Notaries are selected through an [application process](https://github.com/filecoin-project/notary-governance/tree/main/notaries#application--selection-process). If approved, [Root Key Holders](https://github.com/filecoin-project/notary-governance/tree/main/root-key-holders#overview) (executors of the decisions made by the community on-chain) grant Notary status and DataCap amounts. Those interested in becoming Notaries should apply to this role by filing an Issue in the [Notary Governance Repo](https://github.com/filecoin-project/notary-governance/).
+
+### Client
+
+Clients can use their DataCap to incentivize miners to serve their needs - this can include providing additional features and levels of services that meet their specific requirements. In doing so, storage-related goods and services on Filecoin are made more valuable and competitive over time. Notaries vet clients to ensure the client receives DataCap commensurate with their reputation and needs and that the client responsibly allocates that DataCap.
+
+## Using DataCap
+
+### Get DataCap
+
+Clients are required to have an on-chain Filecoin address where DataCap can be received. If you are setting up a new address, make sure to initialize it by sending a minimal amount of FILto it (purchasing some FIL from an exchange, for example).
+
+_Note: As of now, DataCap allocations are a single-use credit on a Filecoin address. If you receive an allocation and require more, you should make a new request with a new address that you have initialized like above. [FIP-0012](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0012.md) was recently accepted and, once implemented, will allow addresses to receive DataCap multiple times._
+
+Clients get DataCap by making a request to a Notary.
+Head over to the [Filecoin Plus Registry](https://plus.fil.org/), and proceed with **For Clients**
+Click **Get Verified**
+Click on **Automatic Verification**. This will take you to an automated Notary, which should be able to provide you with a small amount of DataCap to get you started. https://verify.glif.io/ is an example of such a Notary, which is currently giving 32GiB of DataCap once every 30 days to clients who authenticate themselves with a GitHub account greater than 180 days old.
+
+For DataCap requirements greater than 32GiB - choose **General Verification** in Step 3 above instead. This will take you to the Notary Registry, where you can request DataCap to a specific Notary. Notaries may specialize in the types of requests they'll choose to support - it is recommended that you select a Notary in your region of operation that also covers the general category of *Use Case* you would classify yourself under.
```

Comment:
If you choose not to number the above 3 - this will have to be rephrased since it refers to a "Step 3 above"